src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -19,7 +20,11 @@
     'customHeaders': 'HEADERS',
     'limit': 'LIMIT',
     'currentValue': 'CURRENT VALUE',
-    'unit': 'UNIT'
+    'unit': 'UNIT',
+    'buildId': "BUILDID",
```

Comment:
`BUILD ID`, I thought we want to show build name?

Reply:
https://msazure.visualstudio.com/AzureContainerRegistry/_workitems/edit/2309348 I believe we want build Id rather than build name.